### PR TITLE
CRTX-121331 GCP Google Cloud Logging XDM Modeling

### PR DESCRIPTION
**Pull request by cweltPA**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [CRTX-121331](https://jira-dc.paloaltonetworks.com/browse/CRTX-121331)

## Description
This PR adds XDM mappings to the existing Google Cloud Logging pack. 
The modeling covers mappings for the following log types - 
- [DNS Queries](https://cloud.google.com/dns/docs/monitoring#dns-log-record-format)
- [Audit Logs](https://cloud.google.com/logging/docs/audit)
    -  [Admin Activity audit logs](https://cloud.google.com/logging/docs/audit#admin-activity)
    -  [Data Access audit logs](https://cloud.google.com/logging/docs/audit#data-access)
    -  [System Event audit logs](https://cloud.google.com/logging/docs/audit#system-event)
    -  [Policy Denied audit logs](https://cloud.google.com/logging/docs/audit#policy_denied)

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/GoogleCloudLogging/README.md [hunk 1]
```diff
@@ -1,6 +1,22 @@
-Google Cloud Logging is a service provided by Google Cloud Platform (GCP) that allows you to store, search, analyze, monitor, and alert on log data and events from your applications and services running on GCP.
-The Google Cloud Logging Cortex XSOAR pack helps users to centralize all their logs in a single location, making it easier to troubleshoot issues and gain insights from their data.
 ## What does this pack do
-This integration enables you to:
 
-- Retrieve log entries that originated from a project/folder/organization/billing account.
+The Google Cloud Logging Cortex <~XSOAR>XSOAR</~XSOAR><~XSIAM> <~XSIAM>XSIAM</~XSIAM> pack helps users to centralize all their GCP logs in a single location, making it easier to troubleshoot issues and gain insights from their data:
```
Comment (ShirleyDenkberg) [inline Packs/GoogleCloudLogging/README.md:3]:
```suggestion
The Google Cloud Logging Cortex <~XSOAR>XSOAR</~XSOAR><~XSIAM>XSIAM</~XSIAM> pack helps users to centralize all their GCP logs in a single location, making it easier to troubleshoot issues and gain insights from their data: 
```

**Comment by ShirleyDenkberg** [comment]
@OBavly Doc review completed.

File: Packs/GoogleCloudLogging/README.md (hunk 1 above)
Comment (cweltPA) [reply Packs/GoogleCloudLogging/README.md]: @ShirleyDenkberg Thanks for the quick review. 
I inserted some reorganization and reformatting of the sections to better emphasize the separation between the Integration to the Modeling Rule & Parsing Rule which are independent of the integration. If you could re-review the updated version as well.

**Review by OBavly — APPROVED** [review]

File: Packs/GoogleCloudLogging/README.md (hunk 1 above)
Comment (cweltPA) [reply Packs/GoogleCloudLogging/README.md]: @ShirleyDenkberg Thanks for the quick review. 
I inserted some reorganization and reformatting of the sections on the README and release notes to better emphasize the separation between the Integration to the Modeling Rule & Parsing Rule which are independent of the integration. If you could re-review the updated version as well.

File: Packs/GoogleCloudLogging/README.md [hunk 2]
```diff
@@ -1,6 +1,25 @@
...
+
+### Google Cloud Logging Integration
+The *Google Cloud Logging Integration* enables you to retrieve selected log entries that originated from a project/folder/organization/billing account. See the [Google Cloud Logging integration documentation](https://xsoar.pan.dev/docs/reference/integrations/google-cloud-logging) for additional details.   
+
+
+<~XSIAM>
+### Google Cloud Logging SIEM Content
+The SIEM content includes *Cortex Data Modeling (XDM) Rules* and *Parsing Rules* which are applied on [*Google Cloud Audit Logs*](https://cloud.google.com/logging/docs/audit) and [*Google Cloud DNS Query Logs*](https://cloud.google.com/dns/docs/monitoring#dns-log-record-format) that are ingested into the *`google_cloud_logging_raw`* and *`google_dns_raw`* datasets (respectively) via the *Google Cloud Platform Pub/Sub* data source on Cortex XSIAM. See [*Ingest Logs and Data from a GCP Pub/Sub*](https://docs-cortex.paloaltonetworks.com/r/Cortex-XSIAM/Cortex-XSIAM-Administrator-Guide/Ingest-Logs-and-Data-from-a-GCP-Pub/Sub?tocId=q2ZnSo90ZmO~6aLgts8g5g) for additional details.
```
Comment (ShirleyDenkberg) [inline Packs/GoogleCloudLogging/README.md:11]:
```suggestion
The SIEM content includes *Cortex Data Modeling (XDM) Rules* and *Parsing Rules* which are applied on [*Google Cloud Audit Logs*](https://cloud.google.com/logging/docs/audit) and [*Google Cloud DNS Query Logs*](https://cloud.google.com/dns/docs/monitoring#dns-log-record-format) that are ingested into the *`google_cloud_logging_raw`* and *`google_dns_raw`* datasets (respectively) via the *Google Cloud Platform Pub/Sub* data source on Cortex XSIAM. See [*Ingest Logs and Data from a GCP Pub/Sub*](https://docs-cortex.paloaltonetworks.com/r/Cortex-XSIAM/Cortex-XSIAM-Documentation/Ingest-logs-and-data-from-a-GCP-Pub/Sub?tocId=RyZP~~W~YWbOPGKAClIQHg) for additional details.
```

File: Packs/GoogleCloudLogging/README.md [hunk 3]
```diff
@@ -1,6 +1,25 @@
...
+
+#### Remarks
+- When configuring a [sink](https://cloud.google.com/logging/docs/export/configure_export_v2#creating_sink) to route Google Cloud logs to the [Pub/Sub service](https://cloud.google.com/pubsub/docs/pubsub-basics) as described [here](https://docs-cortex.paloaltonetworks.com/r/Cortex-XSIAM/Cortex-XSIAM-Administrator-Guide/Ingest-Logs-and-Data-from-a-GCP-Pub/Sub?tocId=q2ZnSo90ZmO~6aLgts8g5g), you may wish to create an inclusion filter to include only a subset of the logs. See filter examples [here](https://cloud.google.com/logging/docs/export/configure_export_v2#filter-examples) and samples below:
```
Comment (ShirleyDenkberg) [inline Packs/GoogleCloudLogging/README.md:14]:
```suggestion
- When configuring a [sink](https://cloud.google.com/logging/docs/export/configure_export_v2#creating_sink) to route Google Cloud logs to the [Pub/Sub service](https://cloud.google.com/pubsub/docs/pubsub-basics) as described [here](https://docs-cortex.paloaltonetworks.com/r/Cortex-XSIAM/Cortex-XSIAM-Documentation/Ingest-logs-and-data-from-a-GCP-Pub/Sub?tocId=RyZP~~W~YWbOPGKAClIQHg), you may wish to create an inclusion filter to include only a subset of the logs. See filter examples [here](https://cloud.google.com/logging/docs/export/configure_export_v2#filter-examples) and samples below: 
```

**Comment by ShirleyDenkberg** [comment]
@cweltPA Re-reviewed readme and release notes as requested.

**Pull request merged**